openstack/octavia/templates/octavia-worker-deployment.yaml

```diff
@@ -21,7 +21,9 @@ spec:
       labels:
         app.kubernetes.io/name: octavia-worker-{{ $name }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-        f5-device-model: {{ $v.model }} # used by Limes for capacity discovery (via Prometheus metric "kube_pod_labels")
+        # Labels used by Limes for capacity discovery (via Prometheus metric "kube_pod_labels")
+        f5-device-model: {{ $v.model }}
+        availability-zone: {{ $v.availability_zone }}
```

Comment:
I would like for this label to always have a value. What does it mean for a worker to not be attached to a specific AZ? If it means that we actually do not know which AZ it belongs to, I would like to have:
```go
        availability-zone: {{ $v.availability_zone | default "unknown" }}
```
If it means that it is used only for non-AZ-aware loadbalancers, I would like to have:
```go
        availability-zone: {{ $v.availability_zone | default "any" }}
```
Ref: Enum in <https://pkg.go.dev/github.com/sapcc/go-api-declarations/limes#AvailabilityZone>

Reply:
Every worker is assigned to one and only one F5 pair. Every F5 pair is either assigned to an AZ or cross-AZ. Therefore I will go with the "any" option.